### PR TITLE
Oleksii/ob 300 mp3 tagger

### PR DESCRIPTION
**Pull request by alexone55**

### CODE REVIEW

File: Alexey/Files/mp3_tagger/main/mp3_tagger.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import eyed3
+
+
+def main():
+    file_name = str(input('Enter file name: ')) + '.mp3'
+    answer = str(input('Enter what you want to do: '))
+    audio = eyed3.load(file_name)
+    if answer == 'edit':
```
Comment (glock3) [inline Alexey/Files/mp3_tagger/main/mp3_tagger.py:9]: provide 'help' option to get available commands

File: Alexey/Files/mp3_tagger/main/mp3_tagger.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+def add_tags(audio, new_dictionary):
+    try:
+        audio.tag.artist = new_dictionary['artist']
+        audio.tag.album = new_dictionary['album']
+        audio.tag.album_artist = new_dictionary['album artist']
+        audio.tag.title = new_dictionary['title']
+        audio.tag.track_num = new_dictionary['track num']
+        audio.tag.save()
+    except KeyError:
```
Comment (glock3) [inline Alexey/Files/mp3_tagger/main/mp3_tagger.py]: why do you need this except? You still have exception raised

File: Alexey/Files/mp3_tagger/tests/test_mp3_tagger.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+import unittest
+import mock
+import eyed3
+
+
+class TestMp3Tagger(unittest.TestCase):
+
+    def test_input_wrong_mp3_name_and_expect_error_in_tag_dict(self):
+        with self.assertRaises(OSError) as context:
+            self.assertRaises(OSError, tag_dictionary(eyed3.load('file_name')))
```
Comment (glock3) [inline Alexey/Files/mp3_tagger/tests/test_mp3_tagger.py]: You can remove this test. This kind of exception usually not tested

Comment (glock3) [inline Alexey/Files/mp3_tagger/tests/test_mp3_tagger.py]: You are double checking error. Remove one assert

File: Alexey/Files/mp3_tagger/main/mp3_tagger.py [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+        album_artist = str(input('Enter album artist: '))
+        title = str(input('Enter title: '))
+        track_num = str(input('Enter track num: '))
+        add_tags(audio, new_dictionary(artist, album, album_artist, title, track_num))
+
+    elif answer == 'check':
+        print(tag_dictionary(audio))
+
+
+def new_dictionary(artist, album, album_artist, title, track_num):
```
Comment (glock3) [inline Alexey/Files/mp3_tagger/main/mp3_tagger.py]: Please remove this one-line function

File: Alexey/Files/mp3_tagger/main/mp3_tagger.py [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+    tags_dict = {'artist': artist,
+                 'album': album,
+                 'album artist': album_artist,
+                 'title': title,
+                 'track num': track_num}
+    return tags_dict
+
+
+def tag_dictionary(audio):
```
Comment (glock3) [inline Alexey/Files/mp3_tagger/main/mp3_tagger.py]: Please remove this one line function

File: Alexey/Files/mp3_tagger/tests/test_mp3_tagger.py [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+        album_artist = 'Михаил круг'
+        title = 'Михаил Круг'
+        track_num = 1
+        expect_set = {'artist': 'Михаил Круг', 'album': 'Михаил Круг', 'album artist': 'Михаил круг', 'title': 'Михаил Круг', 'track num': 1}
+        actual_set = new_dictionary(artist, album, album_artist, title, track_num)
+        self.assertEqual(expect_set, actual_set)
+
+    def test_add_new_tags_to_mp3_files(self):
+        actual_set = tag_dictionary(eyed3.load('Михаил Круг - Владимирский Централ.mp3'))
+        add_tags(eyed3.load('Михаил Круг - Владимирский Централ.mp3'), {'artist': 'test', 'album': 'test', 'album artist': 'test', 'title': 'test', 'track num': 1})
```
Comment (glock3) [inline Alexey/Files/mp3_tagger/tests/test_mp3_tagger.py]: Add more tests for add_tags function
Also move test file to new folder with the name 'test-resources'
Add more test files with different tags and amount of tags

File: Alexey/Files/mp3_tagger/main/mp3_tagger.py [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+    if answer == 'edit':
+        artist = str(input('Enter artist: '))
+        album = str(input('Enter album: '))
+        album_artist = str(input('Enter album artist: '))
+        title = str(input('Enter title: '))
+        track_num = str(input('Enter track num: '))
+        add_tags(audio, new_dictionary(artist, album, album_artist, title, track_num))
+
+    elif answer == 'check':
+        print(tag_dictionary(audio))
```
Comment (glock3) [inline Alexey/Files/mp3_tagger/main/mp3_tagger.py]: Please don't use print use log instead https://docs.python.org/3/howto/logging.html

**Review by glock3 — CHANGES_REQUESTED** [review]

File: Alexey/Files/mp3_tagger/main/mp3_tagger.py (hunk 1 above)
Comment (alexone55) [reply Alexey/Files/mp3_tagger/main/mp3_tagger.py:9]: done

**Review by glock3 — APPROVED** [review]

**Pull request merged**